crates/rust-fnv/sgx-tests/app/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "app"
+version = "1.0.0"
+authors = ["Baidu"]
+build = "build.rs"
```

Comment:
Remove `authors`, add edition. Same with this: https://github.com/universal-secure-computing-community/crates-sgx/blob/sgx1.1.1/crates/itoa/sgx-tests/app/Cargo.toml